lib/beaker/hypervisor/openstack.rb

```diff
@@ -271,7 +273,8 @@ def provision_parallel
             create_instance_resources(host)
           rescue => e
             # Handle exceptions in the thread
-            puts "Thread #{host} failed with error: #{e.message}"
+            message = "Thread #{host} failed with error: #{e.message}"
+            puts message; $stderr.puts message
```

Comment:
I'd only output it to stderr
```suggestion
            $stderr.puts message
```

Reply:
this has actually been replaced with logger to conform with the standard already used throughout the functions
```
      # Handle exceptions in the thread
      @logger.error "Thread #{host} failed with error: #{e.message}"
```

Reply:
The diff on this looks awful for some reason.  I did a comparison in another editor for clarity